File: CompareList/comparelist.py
#!/bin/python3

a = list(map(int, input().rstrip().split()))
b = list(map(int, input().rstrip().split()))
x = y = 0
for i in range(len(a)):
    if a[i]>b[i]:
        x += 1
    elif b[i] > a[i]:
        y += 1
print([x,y])

File: CompareList/comparelist.rb
a = gets.rstrip.split.map(&:to_i)
b = gets.rstrip.split.map(&:to_i)
x = y = 0
a.each_with_index do |val, index| 
    if val > b[index]
       x += 1
    end
    if val < b[index]
       y += 1
    end
end
p [x,y]